<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Antenna calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="optionsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creditsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Credits</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAntenna_calculator_by_Merituum">
    <property name="title">
     <string>Antenna calculator by Merituum</string>
    </property>
   </widget>
   <addaction name="menuAntenna_calculator_by_Merituum"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
